jobs/src/main/scala/dcos/metronome/queue/QueueService.scala

```diff
@@ -0,0 +1,18 @@
+package dcos.metronome.queue
+
+import mesosphere.marathon.core.launchqueue.LaunchQueue.QueuedTaskInfo
+
+/**
+  * Provides access to the underlying list of tasks in the launch queue.
+  *
+  */
+trait QueueService {
```

Comment:
Queue sounds too generic to me - do you think it makes sense to make the name more specific?